loader/loader.go

```diff
@@ -664,16 +673,16 @@ func resolveVolumePath(volume types.ServiceVolumeConfig, workingDir string, look
 }
 
 // TODO: make this more robust
-func expandUser(path string, lookupEnv template.Mapping) string {
-	if strings.HasPrefix(path, "~") {
+func expandUser(p string, lookupEnv template.Mapping) string {
```

Comment:
sorry but I definitely not a fan of changing `path` to `p` to make a linter happy, we lost to much meaning of the function argument in that case! 
Isn't possible the alias the import instead?

Reply:
Sure!

Reply:
Renamed the import to `paths`